### DISCUSSION TITLE
Bootstrap NPM Starter for v5?

### DISCUSSION THREAD

**Discussion by kanlukasz** [twbs/bootstrap#34385] [Q&A]
Hi,

Where can I find the **official** NPM Starter for v5?
I know about https://github.com/twbs/bootstrap-npm-starter but there is only v4 version 😞

**Comment by XhmikosR** [comment]
It's unrelated to this repo, but we have a WIP PR https://github.com/twbs/bootstrap-npm-starter/pull/49

**Reply by torzak** [reply]
@kanlukasz --> https://github.com/twbs/examples 

@XhmikosR the bootstrap V5 documentation link to the V4 starter repository here
- https://getbootstrap.com/docs/5.2/customize/sass/#variable-defaults 
- https://getbootstrap.com/docs/5.2/getting-started/download/#npm

I think the link in the documentation must be updated to point directly to https://github.com/twbs/examples and not  https://github.com/twbs/bootstrap-npm-starter

**Reply by mdo** [reply]
This has been updated in v5.3.0's docs in `main`. We typically don't retroactively update old docs versions.
